assets/stylesheets/main.css

```diff
@@ -0,0 +1,209 @@
+  border-radius: 0.3rem;
+  height: 2.4rem;
+}
+
+.fle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.link {
+  width: 11.1111%;
+}
+
+.link:hover {
+  cursor: pointer;
+  background-color: #444;
+}
+
+.menu-link {
+  text-decoration: none;
+  color: white;
+  width: 100%;
+  height: 100%;
+}
+
+.search {
+  width: 1.8rem;
+}
+
+.bord {
+  border-left: 1px solid #555;
+  height: 100%;
+}
+
+.br {
+  border-radius: 25px;
+  background: url(../images/search.jpg) no-repeat right 7px center;
+  background-size: 1.4rem 1.4rem;
+  background-position: left center;
+  width: 80%;
+  height: 1.4rem;
+}
+
+/** main content **/
+
+.section {
+  text-align: center;
+}
+
+.l {
+  text-align: center;
+}
+
+.section .l {
+  text-decoration: none;
+  font-family: 'Barlow Condensed', sans-serif;
+  font-weight: 100;
+  color: rgba(255, 255, 255, 0.755);
+}
+
+.text1 {
+  margin-top: 3.6rem;
+}
+
+.text3 {
+  margin-top: 1.5rem;
+}
+
+.all {
+  display: grid;
+  grid-template-columns: auto auto auto auto auto;
+  max-width: 120rem;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.measure {
+  width: 30rem;
+  height: 16.4835rem;
+}
+
+.all .one {
+  grid-column-start: 1;
+  grid-column-end: 6;
```

Comment:
1) **[Optional]** First, we will have to change the number of columns, we can change it to 8 columns using `grid-column-end: 9;`